modules/ROOT/pages/administration/access-control/manage-roles.adoc

```diff
@@ -706,6 +706,9 @@ SHOW USERS
 5+a|Rows: 5
 |===
 
+Common errors such as attempting to grant roles to users who already have the role will lead to notifications.
```

Comment:
We added notifications now, since we didn't want to introduce the (breaking) changes to revoke commands. 

Please correct me if I am wrong @Hunterness